details/java-jvm.md

```diff
@@ -208,6 +208,7 @@ MLPMNist using DL4J: [![MLPMNist using DL4J](https://img.shields.io/docker/pulls
 
   - [Clojure related ML links](https://github.com/josephmisiti/awesome-machine-learning#clojure)
   - [Scicloj](https://scicloj.github.io/) - an open, free and dynamic hub aimed at advancing doing data science, machine learning and numerical computing in Clojure | [GitHub](https://github.com/scicloj) | [Zulip](https://clojurians.zulipchat.com/#narrow/stream/151924-data-science) | [Twitter](https://twitter.com/scicloj) | [Youtube](http://yt.vu/+scicloj) | [scicloj@gmail.com](mailto:scicloj@gmail.com)
+  - [Practicalli Data Science](https://www.youtube.com/playlist?list=PLpr9V-R8ZxiDUXIR2z8Y8wvhpoPyl0t_D) - data capture, transformation and visualisation - YouTube playlist
```

Comment:
Feel free to add this to other sections of the repo as well, for eg. these two are good candidates of your work:

- https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/details/visualisation.md
- https://github.com/neomatrix369/awesome-ai-ml-dl/tree/master/data (this one has multiple places as well for such resources)

Reply:
I will take a look at the weekend and add some more links.  Thank you.